history/management/commands/run_bot.py

```diff
@@ -155,26 +160,54 @@ def version(message):
                 "More info @ https://github.com/owocki/slack_gamebot " 
             message.reply(help_message)
 
+        def get_active_season(gamename,seasoned):
+            range_start_date = default_start
+
+            try:
+                active_season = Season.objects.get(gamename=gamename,active=True)
+            except Exception as e:
+                active_season = Season.objects.create(gamename=gamename,start_on = range_start_date,active = True)
+                active_season.save()
+
+            if seasoned:
+                range_start_date = active_season.start_on
+            else:
+                active_season = None
+
+            return active_season, range_start_date
+
+        @listen_to('^gamebot alltime leaderboard (.*)',re.IGNORECASE)
+        @listen_to('^gb alltime leaderboard (.*)', re.IGNORECASE)
+        @listen_to('^alltime leaderboard (.*)',re.IGNORECASE)
+        def unseasoned_leaderboard(message,gamename):
+            return _leaderboard(message,gamename,False)
+
         @listen_to('^gamebot leaderboard (.*)',re.IGNORECASE)
         @listen_to('^gb leaderboard (.*)', re.IGNORECASE)
         @listen_to('^leaderboard (.*)',re.IGNORECASE)
-        def leaderboard(message,gamename):
+        def seasoned_leaderboard(message,gamename):
+            return _leaderboard(message,gamename,True)
+
+        def _leaderboard(message,gamename,seasoned=False):
             #input sanitization
             gamename = gamename.strip()
```

Comment:
Sanitization should make it lowercase as well. Could this be a quick change in this PR? That way we could avoid recording Chess and chess and chesS. I'm pretty sure this isn't implemented yet, though I could be wrong.